### ISSUE TITLE
nameko.standalone.__init__.py doc strings errors

### ISSUE THREAD

**Issue by litnimax** [nameko/nameko#661]
Hi!
This does not work:
```
>>> from nameko.standalone.rpc import rpc_proxy
```
Seems the doc strings are outdated.

**Comment by mattbennett** [comment]
Yes, they are out of date I'm afraid. A PR would be great if you'd like to fix them ;)

Note that we have a [v3 pre-release branch](https://github.com/nameko/nameko/tree/v3.0.0-rc) (where they also need updating) -- so if you are able to contribute a PR it would be best against that branch.
